src/managed/CommonManaged.props

```diff
@@ -5,7 +5,7 @@
 
   <PropertyGroup>
     <RepoRoot>$([System.IO.Path]::GetFullPath('$(MSBuildThisFileDirectory)../..'))/</RepoRoot>
-    <VersionPrefix>2.0.0</VersionPrefix>
+    <VersionPrefix>$(VersionPrefix)</VersionPrefix>
```

Comment:
hmm... that's not what I have on my machine.  I have 

```
    <VersionPrefix>2.0.0</VersionPrefix>
    <AssemblyFileVersion>$(VersionPrefix)</AssemblyFileVersion>
```
I'll try to figure out why this diff is weird